t/t1017-update-db.t

```diff
@@ -96,70 +87,15 @@ test_expect_success 'get all the columns from the queue_table and make sure the
 	test_cmp queue_table_columns.expected queue_table_columns.test
 '
 
-test_expect_success 'successfully update flux-accounting DB from v0.10.0' '
-	cp ${DB_v0_10_0} tmp_v0_10_0.db &&
-	chmod 666 tmp_v0_10_0.db &&
-	flux account-update-db -p tmp_v0_10_0.db
-'
-
-test_expect_success 'successfully update flux-accounting DB from v0.11.0' '
-	cp ${DB_v0_11_0} tmp_v0_11_0.db &&
-	chmod 666 tmp_v0_11_0.db &&
-	flux account-update-db -p tmp_v0_11_0.db
-'
-
-test_expect_success 'successfully update flux-accounting DB from v0.12.0' '
-	cp ${DB_v0_12_0} tmp_v0_12_0.db &&
-	chmod 666 tmp_v0_12_0.db &&
-	flux account-update-db -p tmp_v0_12_0.db
-'
-
-test_expect_success 'successfully update flux-accounting DB from v0.13.0' '
-	cp ${DB_v0_13_0} tmp_v0_13_0.db &&
-	chmod 666 tmp_v0_13_0.db &&
-	flux account-update-db -p tmp_v0_13_0.db
-'
-
-test_expect_success 'successfully update flux-accounting DB from v0.14.0' '
-	cp ${DB_v0_14_0} tmp_v0_14_0.db &&
-	chmod 666 tmp_v0_14_0.db &&
-	flux account-update-db -p tmp_v0_14_0.db
-'
-
-test_expect_success 'successfully update flux-accounting DB from v0.15.0' '
-	cp ${DB_v0_15_0} tmp_v0_15_0.db &&
-	chmod 666 tmp_v0_15_0.db &&
-	flux account-update-db -p tmp_v0_15_0.db
-'
-
-test_expect_success 'successfully update flux-accounting DB from v0.16.0' '
-	cp ${DB_v0_16_0} tmp_v0_16_0.db &&
-	chmod 666 tmp_v0_16_0.db &&
-	flux account-update-db -p tmp_v0_16_0.db
-'
-
-test_expect_success 'successfully update flux-accounting DB from v0.17.0' '
-	cp ${DB_v0_17_0} tmp_v0_17_0.db &&
-	chmod 666 tmp_v0_17_0.db &&
-	flux account-update-db -p tmp_v0_17_0.db
-'
-
-test_expect_success 'successfully update flux-accounting DB from v0.18.0' '
-	cp ${DB_v0_18_0} tmp_v0_18_0.db &&
-	chmod 666 tmp_v0_18_0.db &&
-	flux account-update-db -p tmp_v0_18_0.db
-'
-
-test_expect_success 'remove temporary test DBs' '
-	rm tmp_v0_10_0.db &&
-	rm tmp_v0_11_0.db &&
-	rm tmp_v0_12_0.db &&
-	rm tmp_v0_13_0.db &&
-	rm tmp_v0_14_0.db &&
-	rm tmp_v0_15_0.db &&
-	rm tmp_v0_16_0.db &&
-	rm tmp_v0_17_0.db &&
-	rm tmp_v0_18_0.db
-'
+# test updating flux-accounting databases from older versions,
+# starting with v0.10.0
+for db in ${SHARNESS_TEST_SRCDIR}/expected/test_dbs/*; do
+	if [[ $db == *FluxAccounting* ]]; then
+		cp $db $(basename $db | cut -d '.' -f 1)_tmp.db
+		tmp_db=$(basename $db | cut -d '.' -f 1)_tmp.db
```

Comment:
Set `tmp_db` first then you can use that variable in the `cp`:
```suggestion
		tmp_db=$(basename $db | cut -d '.' -f 1)_tmp.db
		cp $db $tmp_db
```

Reply:
Oh, duh! I don't know why I didn't do this. Thanks for pointing this out, I'll push up a fix.